internal/authtoken/repository_test.go

```diff
@@ -63,10 +65,12 @@ func TestRepository_New(t *testing.T) {
 				opts: []Option{WithLimit(5)},
 			},
 			want: &Repository{
-				reader:       rw,
-				writer:       rw,
-				kms:          kmsCache,
-				defaultLimit: 5,
+				reader:              rw,
+				writer:              rw,
+				kms:                 kmsCache,
+				defaultLimit:        5,
+				timeToLiveDuration:  7 * 24 * time.Hour,
+				timeToStaleDuration: 24 * time.Hour,
```

Comment:
Should add a non-exported constant for these default values in options.go and then use that constant here.